user/multi-cpu-architectures.md

```diff
@@ -35,28 +57,29 @@ arch:
   - amd64
   - ppc64le
   - s390x
-  - arm64
+  - arm64  # please note arm64-graviton2 requires explicit virt: [lxd|vm] tag so it's recommended for jobs.include, see below
 os: linux  # different CPU architectures are only supported on Linux
 ```
 {: data-file=".travis.yml"}
 
 If you are already using a [build matrix](/user/customizing-the-build/#build-matrix) to test multiple versions, the `arch` key also multiplies the matrix.
 
-- The ppc64le (IBM Power) and s390x (IBM Z) build jobs are run in an LXD compliant Linux OS image. 
-- The arm64 CPU architecture build job is run in an LXD compliant Linux OS image.
+- The `ppc64le` (IBM Power) and `s390x` (IBM Z) build jobs are run in an LXD compliant Linux OS image. 
+- The `arm64` CPU architecture build job is run in an LXD compliant Linux OS image.
+- The `arm64-graviton2` architecture builds can be run on both LXD and regular 'full VM' environments. You **need** explicitely set target environment by using `virt` key. A `virt: vm` routes build jobs to full virtual machine setup while `virt: lxd` routes build jobs to LXD container setup. 
```

Comment:
You need **to** explicitly set **the** target environment by using the 'virt' key. A `virt: vm` routes build jobs to **a** full virtual machine setup while `virt: lxd` routes build jobs to **an** LXD container setup.

Reply:
Updating on master, thanks!